etc/config.system.toml

```
@@ -0,0 +1,30 @@
+# Configuring Cascade for system-wide use
+# =======================================
+#
+# This configuration file is packaged with Cascade for system-wide setups (e.g.
+# with systemd).  See 'config.template.toml' for a complete description of the
+# settings and their defaults.
+
+version = "v1"
+
+policy-dir = "/etc/cascade/policies"
+zone-state-dir = "/var/lib/cascade/zone-state"
+tsig-store-path = "/var/lib/cascade/tsig-keys.db"
+
+[daemon]
+log-level = "info"
+log-target = { type = "syslog" }
+daemonize = false
+
+[remote-control]
+servers = ["127.0.0.1:8950", "[::1]:8950"] # TODO: Pick a good default port
+
+[loader]
+notif-listeners = ["127.0.0.1:8050", "[::1]:8050"] # TODO: Pick a good default port
```

```suggestion
notify-listeners = ["127.0.0.1:8050", "[::1]:8050"] # TODO: Pick a good default port
```
And don't forget to update this, since the rust side changed